cmd/helm/install.go

```diff
@@ -285,12 +286,32 @@ func (v *values) Set(data string) error {
 	return nil
 }
 
+func typedVal(val string) interface{} {
+	if val == "true" {
+		return true
+	}
+
+	if val == "false" {
+		return false
+	}
+
+	if iv, err := strconv.ParseInt(val, 10, 64); err == nil {
```

Comment:
Technically we may not need to parse integers, since in templating, a string and integer will generally be printed the same. The only case would be if a template function needed an integer to perform integer operations, but I haven't really seen that usecase.